clustering/infinispan/extension/src/main/resources/org/jboss/as/clustering/infinispan/subsystem/LocalDescriptions.properties

```diff
@@ -357,6 +357,7 @@ infinispan.backup.bring-site-online=Re-enables a previously disabled backup site
 infinispan.backup.take-site-offline=Disables backup to a remote site.
 
 infinispan.component.backup-for=A cache for which this cache acts as a backup (for use with cross site replication).
+infinispan.component.backup-for.deprecated=Deprecated. Backup designation should match the current cache name.
```

Comment:
s/should/have to?

Reply:
They are synonymous.

Reply:
Hm, I mean "must" then.

https://www.ietf.org/rfc/rfc2119.txt

Reply:
OK - you win.